sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1192,3 +1192,57 @@ def test_dtype_match():
             lr_64.fit(X_64, y_64)
             assert_equal(lr_64.coef_.dtype, X_64.dtype)
             assert_almost_equal(lr_32.coef_, lr_64.coef_.astype(np.float32))
+
+
+def test_warm_start_converge_LR():
+    # Test to see that the logistic regression converges on warm start,
+    # with multi_class='multinomial'
+    # Set independent variable values
+
+    Z = np.array([
+            [0., 0.],
+            [1.33448632, 0.],
+            [1.48790105, -0.33289528],
+            [-0.47953866, -0.61499779],
+            [1.55548163, 1.14414766],
+            [-0.31476657, -1.29024053],
+            [-1.40220786, -0.26316645],
+            [2.227822, -0.75403668],
+            [-0.78170885, -1.66963585],
+            [2.24057471, -0.74555021],
+            [-1.74809665,  2.25340192],
+            [-1.74958841,  2.2566389],
+            [2.25984734, -1.75106702],
+            [0.50598996, -0.77338402],
+            [1.21968303,  0.57530831],
+            [1.65370219, -0.36647173],
+            [0.66569897,  1.77740068],
+            [-0.37088553, -0.92379819],
+            [-1.17757946, -0.25393047],
+            [-1.624227, 0.71525192]])
+
+    # Set dependant variable values
+    Y = np.array([1, 0, 0, 1, 0, 0, 0, 0,
+                  0, 0, 1, 1, 1, 0, 0, 1,
+                  0, 0, 1, 1], dtype=np.int32)
+    lr1 = LogisticRegression(C=1000000, multi_class='multinomial',
+                             solver='sag', tol=0.0001, warm_start=False,
+                             verbose=0)
+    lr2 = LogisticRegression(C=1000000, multi_class='multinomial',
+                             solver='sag', tol=0.0001, warm_start=True,
+                             verbose=0)
+
+    lr1_loss = []
+    for i in range(5):
```

Comment:
Is the issue only applicable for some random states? If not, I would not bother with repeated trials.

Reply:
This is used to show that even after repeated fitting, the value converges

Reply:
I think we can do it just once in the warm_start=False case.